src/neo/SmartContract/InteropService.Encode.cs

```diff
@@ -0,0 +1,52 @@
+using Neo.Cryptography;
+using Neo.IO;
+using Neo.VM;
+using Neo.Wallets;
+using System;
+using System.Linq;
+
+namespace Neo.SmartContract
+{
+    partial class InteropService
+    {
+        public static class Encode
+        {
+            public static readonly InteropDescriptor PubKeyToAddress = Register("Neo.Encode.PubKeyToAddress", Encode_PubKeyToAddress, 0_00010000, TriggerType.All, CallFlags.None);
```

Comment:
Same public key can be converted to different addresses. I don't think we should have this kind of SYSCALLs.

Reply:
Even through we can use some other contract pattern to build the witnesses scripts which lead to different addresses. But in most case, we are using the default contract pattern. I think we can add some comments here is enough.

Reply:
Hi, I have updated the function annotation so that people will know that this function is only used for account information. Could you please have a look? Besides I believe this function is needed in several projects like Celer & SeraphID...